Inc/ST-LIB_LOW/Sensors/EncoderSensor/EncoderSensor.hpp

```diff
@@ -12,16 +12,16 @@
 #include "ErrorHandler/ErrorHandler.hpp"
 #define COUNTER_DISTANCE_IN_METERS 0.025
 #define NANO_SECOND 1000000000.0
-#define N_FRAMES 5
+#define N_FRAMES 8
 #define FRAME_SIZE_IN_SECONDS 0.01
-#define START_COUNTER 32768
+#define START_COUNTER UINT16_MAX / 2
 #define CLOCK_MAX_VALUE 4294967295 //here goes the tim2 counter period
 
 
 
 class EncoderSensor{
 public:
-	EncoderSensor(Pin pin1, Pin pin2, double* position, double* speed, double* acceleration);
+	EncoderSensor(Pin pin1, Pin pin2, double* position, double* direction, double* speed, double* acceleration);
```

Comment:
```suggestion
	EncoderSensor(Pin& pin1, Pin& pin2, double* position, double* direction, double* speed, double* acceleration);
```

Pines por referencia